themes/docsy/layouts/partials/scripts.html

```diff
@@ -18,4 +18,10 @@
 {{ $js := $js | minify | fingerprint }}
 <script src="{{ $js.RelPermalink }}" integrity="{{ $js.Data.Integrity }}" crossorigin="anonymous"></script>
 {{ end }}
+
+<script src="https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.min.js"></script>
+<script>
+    window.onload = () => mermaid.init(undefined, ".language-mermaid");
+</script>
```

Comment:
@Jtrust I added this to render [mermaid](https://mermaid-js.github.io/mermaid/#/) graphs.